low-level/framework/rabbitmq/rabbitmq_egress_accumulated_msgs_processor.py

```diff
@@ -130,12 +118,11 @@ def run(self):
                         time_diff = int(time.time()) - int(event_time)
                         if time_diff > self.MSG_TIMEOUT:
                             continue
-                    self._connection.publish(exchange=self._exchange_name,routing_key=self._routing_key,properties=msg_props,body=message)
                     if "sensor_response_type" in dict_msg["message"]:
                         logger.info(f"Publishing Accumulated Alert: {message}")
-                self._connection.cleanup()
-        except connection_exceptions as e:
-            logger.error(connection_error_msg.format(e))
+                    self._producer.send([message])
+        except MessageBusError as e:
```

Comment:
This error comes when exactly ? Just in case of any error? Cant we use this to accumulate messages.

Reply:
Should we try to resend?

Reply:
`Cant we use this to accumulate messages.` sensor will be blocked in case of kafka is not available.

Reply:
sensor would block until EOS-17626 is fixed?

Reply:
Yes, if kafka service is stopped for some reason.
I tried stopping kafka and then initialize MessageProducer
It tries to connect to kafka infinitely